server/server.js

```diff
@@ -17,6 +44,42 @@ app.get('/carousel/:gameId', (req, res) => {
   });
 });
 
+// Update
+
+app.put('/put', (req, res) => {
+  let newRecord = {
+    gameTitle: faker.name.title(),
+    newPrice: 59.99,
+    preOwnedPrice: 54.99,
+    digitalPrice: 59.99,
+    mainImage: seed.randomMainImage(),
+    images: seed.randomImages(),
+  }
+  let gameTitle = '';
+  db.updateRecord(gameTitle, param, (err, success) => {
+    if (err) {
+      res.send(404);
+    } else {
+      res.send(200);
+    }
+  });
+  res.end();
+});
+
+// Delete
+
+app.delete('/delete', (req, res) => {
+  const gameTitle = 'game'
```

Comment:
Have you tested this and gotten the expected results?  I'm not seeing how you are identifying which specific record would be deleted. As is, would it be possible for someone to send a delete request for a record of their choosing?

Reply:
It should work. They would just need to send in the title of the game and it should delete that record.